gossipd/gossipd.c

```diff
@@ -1871,17 +1871,20 @@ static struct io_plan *getroute_req(struct io_conn *conn, struct daemon *daemon,
 	u8 *out;
 	struct route_hop *hops;
 	double fuzz;
+	struct short_channel_id *excluded;
+	bool *excluded_dir;
```

Comment:
Not really a boolean, but I get why this is used.

Reply:
Yeah, used int elsewhere, should do so here.